core/DataAccess/LogAggregator.php

```diff
@@ -1160,6 +1159,9 @@ public static function makeArrayOneColumn($row, $columnName, $lookForThisPrefix
 
     public function getDb()
     {
-        return Db::getReader();
+        /** @var ArchivingStatus $archivingStatus */
+        $archivingStatus = StaticContainer::get(ArchivingStatus::class);
+        $archivingLock = $archivingStatus->getCurrentArchivingLock();
+        return new ArchivingDbAdapter(Db::getReader(), $archivingLock, $this->logger);
```

Comment:
@diosmosis is there a chance you forgot to commit this file?

Reply:
a chance? no... I definitely forgot to commit it :) added now